docs/reference/rule_attributes.md

```diff
@@ -0,0 +1,68 @@
+# Ick Toml Reference
+The following attributes can be set in an `ick.toml` or `pyproject.toml`.
+
+## `[[ruleset]]` Attributes
+
+### Required (at least one)
+- `url` (str): A URL to an external repository containing rules
+- `path` (str): A local path to a directory containing rules
+
+### Optional Attributes
+- `prefix` (str): A prefix to use for the rules from this mount. If not specified, it will be derived from the last component of the URL or path.
```

Comment:
```suggestion
- `prefix` (str): A prefix to use for the rules from this ruleset. If not specified, it will be derived from the last component of the URL or path.
```